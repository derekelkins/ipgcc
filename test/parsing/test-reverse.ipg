S -> A[0, B.START] B[0, EOI];
A -> "a"[EOI - 1, EOI];
B -> "b"[EOI - 1, EOI];
