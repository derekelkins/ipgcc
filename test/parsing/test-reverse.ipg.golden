S
  -> B@0[0, EOI]
     A@0[0, B@0.START];

A
  -> "a"[EOI - 1, EOI];

B
  -> "b"[EOI - 1, EOI];